ios/Sources/AboutFeature/AboutScreen.swift

```diff
@@ -11,191 +11,86 @@ public struct AboutScreen: View {
+                    WithViewStore(store) { viewStore in
+                        ErrorView(tapReload: {
+                            viewStore.send(.reload)
+                        })
                     }
                 }
-                .listStyle(PlainListStyle())
-                .padding(.top, 20)
-            }
-        }
-    }
-    func banner(tap: @escaping () -> Void) -> some View {
-        VStack {
-            Spacer()
-
-            HStack(spacing: 8) {
-                Text(L10n.AboutDroidKaigiScreen.whatIs)
-                    .foregroundColor(.white)
-                    .font(.headline)
-
-                AssetImage.logoTitle.image
-                    .colorScheme(.dark)
-                    .frame(height: 22)
-
-                Spacer()
-
-                AssetImage.iconChevron.image
-                    .renderingMode(.template)
-                    .foregroundColor(.white)
-            }
-            .padding(16)
-            .background(AssetColor.dark.color)
-            .cornerRadius(6)
-            .onTapGesture(perform: tap)
-            .padding(16)
+            )
         }
     }
 }
 
-private extension SelectedType {
-    var title: String {
-        switch self {
-        case .staff:
-            return L10n.AboutScreen.Picker.staff
-        case .contributor:
-            return L10n.AboutScreen.Picker.contributor
+private extension AboutAction {
+    init(action: AboutScreen.ViewAction) {
+        switch action {
+        case .progressViewAppeared, .reload:
+            self = .refresh
         }
     }
 }
 
 #if DEBUG
+import Repository
+
 public struct AboutScreen_Previews: PreviewProvider {
     public static var previews: some View {
         Group {
-            AboutScreen(
-                store: .init(
-                    initialState: .init(
-                        staffs: [.mock(), .mock(), .mock()],
-                        contributors: [.mock(), .mock(), .mock()]
-                    ),
-                    reducer: .empty,
-                    environment: {}
-                )
-            )
-            .previewDevice(.init(rawValue: "iPhone 12"))
-            .environment(\.colorScheme, .dark)
-            AboutScreen(
-                store: .init(
-                    initialState: .init(
-                        staffs: [.mock(), .mock(), .mock()],
-                        contributors: [.mock(), .mock(), .mock()]
-                    ),
-                    reducer: .empty,
-                    environment: {}
+            aboutScreen
+                .previewDevice(.init(rawValue: "iPhone 12"))
+                .environment(\.colorScheme, .dark)
+            aboutScreen
+                .previewDevice(.init(rawValue: "iPhone 12"))
+                .environment(\.colorScheme, .light)
+        }
+    }
+
+    static var aboutScreen: some View {
+        AboutScreen(
+            store: Store<AboutState, AboutAction>(
+                initialState: .initialized(.init()),
```

Comment:
I think it's better to add previews with each state not only `needToInitialize`.

Reply:
Add every state preview cases.
I don't know why but it seems that dark mode doesn't affect to preview of errorOccurred state.
It looks in simulator debug build. If you have any idea please let me know.
 d963558990a5e9b93158a74cc18ca2df489c5de9

![preview_error_occured](https://user-images.githubusercontent.com/13119930/135306285-95410c22-50ff-41da-a0f0-7f70aa39573b.png)

Reply:
I will fix this problem in other PR. You can leave this as it is.

Just in case please use `ForEach(ColorScheme.allCases, id: \.self) { colorScheme in }` to make shorten.

Reply:
I found cause of this problem and fixed it. But Ok, I reverted it.
> Just in case please use ForEach(ColorScheme.allCases, id: \.self) { colorScheme in } to make shorten.
I noticed it, thank you :) refactored in 0d7aa92e6ef69304966fdcaaf741a690c17b1d38